packages/millicast-sdk/src/Publish.js

```diff
@@ -255,4 +274,14 @@ export default class Publish extends BaseWebRTC {
       })
     }
   }
+
+  sendMetadata (message, uuid = '6e9cfd2a-5907-49ff-b363-8978a6e8340e') {
```

Comment:
nitpick : Because the users of the SDK might need it to check if the message was published by them and would need to compare, we can expose the UUID out. 


```suggestion jsavascript
export const DOLBY_SEI_UUID = '6e9cfd2a-5907-49ff-b363-8978a6e8340e';
  sendMetadata (message, uuid = DOLBY_SEI_UUID) {
```